typescript/agentkit/src/action-providers/vaultsfyi/schemas.ts

```diff
@@ -0,0 +1,109 @@
+import { z } from "zod";
+
+/**
+ * Action schemas for the vaultsfyi action provider.
+ *
+ * This file contains the Zod schemas that define the shape and validation
+ * rules for action parameters in the vaultsfyi action provider.
+ */
+
+const networkSchema = z.enum([
```

Comment:
Is there a reason this is distinct from the list of supported chains in the constants folder?

If they should be identical lists, can this be derived from it? e.g. `Object.values(supportedChains)`?